adr/0007-choosing-an-open-source-component.md

```diff
@@ -0,0 +1,63 @@
+# Choosing an open source component
+
+Date: 2018-01-09
+
+## Status
+
+Proposed.
+
+## Context 
+
+Using poor quality or unmaintained third-party software can be at the least inconvenient, and at the most downright dangerous: for example, it can fail to take advantage of the benefits from the latest technology, and security holes can go unpatched.
+
+Using third-party software that is difficult to maintain integrated in our environment can make updating it difficult, time-consuming or risky.
+
+Using software with an inappropriate license may present a legal risk, or may risk others choosing not to use our software because of the burden the license subsequently places on them.  
```

Comment:
nit-pick: in the UK license is the verb, licence is the noun.

Reply:
we need an ADR for British vs. American English 🇬🇧 🇺🇸

Reply:
+1

I used to have a yellow sticky note next to me on the wall that would remind me ;) I'm certain I've lapsed since then

Reply:
https://github.com/muan/spiffing

Reply:
https://twitter.com/PaulDJohnston/status/958362583580463107